modules/toolbar.js

```diff
@@ -107,7 +107,7 @@ class Toolbar extends Module {
           option.selected = true;
         }
       } if (input.value) {
-        input.classList.toggle('ql-active', input.value == formats[format]);  // Intentional ==
+        input.classList.toggle('ql-active', input.value === formats[format]);
```

Comment:
since this change allows a white list to be specified as integers instead of strings but the value "+1" actually matters for indent toolbar buttons, it needs to be ===.  

The way it was, an active button can't be clicked twice and the button is determined active if the value of the button is the same as the value of the format. An active button can't be activated again and the value for the button was being set to `false` before being passed through to format. 

The root cause is that == will coerce a string to an integer to do a comparison, which in the case of indent means a delta with format `'indent': 1` has the same value as `"+1"`, which is not really true for the `indent "+1"` button.

I think this change isn't necessary if the restriction that whitelists be strings and and Attributor.value always returns a string is in place.  If we allow the whitelist to contain numbers, then the attributor with a numeric whitelist's value function needs to return a number.  If value can return a number instead of string then toolbar update needs strict ===.